src/platform/OpenThread/GenericNetworkCommissioningThreadDriver.cpp

```diff
@@ -245,6 +245,24 @@ bool GenericThreadDriver::ThreadNetworkIterator::Next(Network & item)
     return true;
 }
 
+ThreadCapabilities GenericThreadDriver::GetSupportedThreadFeatures()
+{
+    BitMask<ThreadCapabilities> capabilites = 0;
+    capabilites.SetField(ThreadCapabilities::kIsBorderRouterCapable,
+                         CHIP_DEVICE_CONFIG_THREAD_BORDER_ROUTER /*OPENTHREAD_CONFIG_BORDER_ROUTER_ENABLE*/);
```

Comment:
same here (and maybe in other places) - this comment is unclear to me in what it means and how it helps reading the code.